### DISCUSSION TITLE
Set viseron to record whole stream?

### DISCUSSION THREAD

**Discussion by Artuz37** [roflcoopter/viseron#364] [Q&A]
Hi, in my usecase I would ideally run two instances of Viseron, one which would store videos where some movement was detected, and second one which stores everything. Can it be done somehow?

**Comment by roflcoopter** [comment]
You might be able to get creative with the FFmpeg config to achieve this, but right now there is no way of seeing it in the UI.
Are you interested in getting it working anyway? I could have a look to see if i can get a working config example

**Reply by Artuz37** [reply]
Hi! Yeah, I am very much interested if you could manage to come up with such a config, I was trying myself but with no success

**Reply by roflcoopter** [reply]
Sadly I didn't succeed with the current config options.
Would need to add it as a feature. Could you open a feature request for it?
